### PR TITLE
YONK-687: API that returns a list of users who have passed the course.

### PR DESCRIPTION
**Pull request by aamishbaloch**

### CODE REVIEW

File: edx_solutions_api_integration/courses/tests.py [hunk 1]
```diff
@@ -1225,6 +1225,42 @@ def test_courses_users_list_courses_enrolled(self):
...
+
+        # create course enrollments
+        CourseEnrollmentFactory.create(user=users[0], course_id=course.id)
+        CourseEnrollmentFactory.create(user=users[0], course_id=course2.id)
+        CourseEnrollmentFactory.create(user=users[1], course_id=course2.id)
+
+        user_grade, user_proforma_grade = 0.9, 0.91
+
+        # create student gradebook
+        StudentGradebook.objects.create(user=users[0], course_id=course.id,
```
Comment (ziafazal) [inline edx_solutions_api_integration/courses/tests.py]: We should use score publish events to save grades.

File: edx_solutions_api_integration/courses/views.py [hunk 2]
```diff
@@ -1124,6 +1124,50 @@ def get_queryset(self):
         return users
 
 
+class CoursesUsersPassedList(SecureAPIView):
+    """
+    **Use Case**
+
+        CoursesUsersPassedList returns a collection of users passed in the course.
+
+        You also use CoursesUsersPassedList to enroll a new user in the course.
```
Comment (ziafazal) [inline edx_solutions_api_integration/courses/views.py]: I think we should remove this in docstring.

File: edx_solutions_api_integration/courses/views.py [hunk 3]
```diff
@@ -1124,6 +1124,50 @@ def get_queryset(self):
         return users
 
 
+class CoursesUsersPassedList(SecureAPIView):
+    """
+    **Use Case**
+
+        CoursesUsersPassedList returns a collection of users passed in the course.
```
Comment (ziafazal) [inline edx_solutions_api_integration/courses/views.py]: I think we could have more appropriate return statement here something like this `CoursesUsersPassedList returns a list of user ids passed in the course`

File: edx_solutions_api_integration/courses/views.py [hunk 4]
```diff
@@ -1124,6 +1124,50 @@ def get_queryset(self):
...
+
+        You also use CoursesUsersPassedList to enroll a new user in the course.
+
+    **Example Requests**
+
+          GET /api/courses/{course_id}/users/passed
+
+    **GET Response Values**
+
+        * results: The collection of user_ids in the course.
```
Comment (ziafazal) [inline edx_solutions_api_integration/courses/views.py]: Need to update uri of the API in docstring

File: edx_solutions_api_integration/courses/views.py [hunk 5]
```diff
@@ -1124,6 +1124,50 @@ def get_queryset(self):
...
+        * GET supports filtering of user by organization(s), groups
+         * To get users paased in a course and are also member of organization
+         ```/api/courses/{course_id}/users?organizations={organization_id}```
+         * organizations filter can be a single id or multiple ids separated by comma
+         ```/api/courses/{course_id}/users?organizations={organization_id1},{organization_id2}```
+         * To get users paased in a course and also member of specific groups
+         ```/api/courses/{course_id}/users?groups={group_id1},{group_id2}```
+    """
+
+    def get(self, request, course_id):  # pylint: disable=W0613
```
Comment (ziafazal) [inline edx_solutions_api_integration/courses/views.py]: nit: use pylint disable statement instead of code. i.e `# pylint: disable=unused-argument`

**Comment by aamishbaloch** [comment]
@ziafazal done with the changes you mentioned above. Please review.

**Pull request merged**